obspy/imaging/scripts/scan.py

```diff
@@ -434,7 +434,11 @@ def plot(self, outfile=None, show=True, fig=None, plot_x=True,
             width = min(width, height * 4)
             fig.set_figwidth(width)
             plt.subplots_adjust(top=1, bottom=0, left=0, right=1)
-            plt.tight_layout()
```

Comment:
Why do we call `tight_layout` on line 425 too? It seems enough to do it once after setting both width and height?

Reply:
fixed, thanks!